### PR TITLE
clean up poms

### LINKED ISSUE #184: in 4.0.3: there are still imports from org.apache.commons.lang here and there (in Head.java for instance)
Hi,

Running a project without any dependency pooling commons-lang, I was facing an exception:

[2020-09-28 08:52:30] [info] Caused by: java.lang.NoClassDefFoundError: org/apache/commons/lang/StringUtils
[2020-09-28 08:52:30] [info] #011at com.jgeppert.struts2.jquery.components.Head.evaluateParams(Head.java:196) ~[struts2-jquery-plugin-4.0.3.jar:?]
[2020-09-28 08:52:30] [info] #011at org.apache.struts2.components.UIBean.end(UIBean.java:533) ~[struts2-core-2.5.22.jar:2.5.22]
[2020-09-28 08:52:30] [info] #011at org.apache.struts2.views.jsp.ComponentTagSupport.doEndTag(ComponentTagSupport.java:39) ~[struts2-core-2.5.22.jar:2.5.22]
...
 
So here is the result of grep -rn "\.commons\.lang\." :

struts2-jquery-plugin/src/main/java/com/jgeppert/struts2/jquery/components/Head.java:24:import org.apache.commons.lang.StringUtils;
struts2-jquery-richtext-plugin/src/main/java/com/jgeppert/struts2/jquery/richtext/actions/CkeditorUpload.java:6:import org.apache.commons.lang.ArrayUtils;
struts2-jquery-showcase/src/main/java/com/jgeppert/struts2/jquery/showcase/Autocompleter.java:25:import org.apache.commons.lang.StringUtils;

Thanks

### LINKED ISSUE #216: cleanup poms

### LINKED ISSUE #235: update spring version to 5.3.21

### LINKED ISSUE #236: update hibernate to version 5.6.9.Final

### LINKED ISSUE #237: update struts-annotations version to 1.0.7

### PR DESCRIPTION
**Pull request by sdutry**
issue: #216 

cleaned up the poms of the projects a bit.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -84,14 +85,20 @@
...
 		<!-- dependency versions -->
 		<struts2.version>6.0.0</struts2.version>
 		<tlib.version>2.3</tlib.version>
 		<minify.version>1.7.4</minify.version>
 		<beanutils.version>1.9.4</beanutils.version>
 		<log4j2.version>2.17.2</log4j2.version>
-		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<jstl.version>1.1.2</jstl.version>
+		<taglibs.standard.version>1.1.2</taglibs.standard.version>
+		<struts-annotations.version>1.0.6</struts-annotations.version>
```
Comment (lukaszlenart) [inline pom.xml]: 1.0.7 is already available
https://github.com/apache/struts-annotations/releases/tag/STRUTS_ANNOTATIONS_1_0_7

Comment (sdutry) [reply pom.xml]: Updated.

Not all version updates for dependencies are done with this PR, it's just a small restructure of the POM - files and i added some of the version updates ( which should now be a bit easier to do)

Comment (lukaszlenart) [reply pom.xml]: Sure, I'm fine with creating a ticket and updating this dependency with another PR.

**Comment by lukaszlenart** [comment]
LGTM 👍

**Pull request merged**